tests/errata/test_errata_vmis.py

```diff
@@ -0,0 +1,118 @@
+import os
+from mock import patch
+
+import pytest
+
+from pushsource import Source, VMIPushItem, AmiPushItem, VHDPushItem
+
+
+def test_errata_ami_via_koji(fake_errata_tool, fake_koji, koji_dir):
+    """Errata source containing a module yields modules & RPMs taken
+    from koji source"""
+
+    source = Source.get(
+        "errata:https://errata.example.com",
+        errata="RHSA-2022:78146",
+        koji_source="koji:https://koji.example.com?basedir=%s" % koji_dir,
+    )
+
+    nvr = "rhel-ec2-8.7-1"
+    name = "rhel"
+    version = "8.7"
+    release = "1"
+
+    # Insert some data
+    fake_koji.build_data[1234] = {
+        "id": 1234,
+        "name": name,
+        "version": version,
+        "release": release,
+        "nvr": nvr,
+        "completion_time": "2022-12-20 12:30:25",
+        "extra": {"typeinfo": {"image": {}}},
+    }
+    fake_koji.build_data[nvr] = fake_koji.build_data[1234]
+
+    imgs = {
+        "foo-1.0-20230110.sp.22.raw.xz": "raw-xz",
+    }
+
+    archives = []
+    for fname, tname in imgs.items():
+        archives.append(
+            {
+                "btype": "image",
+                "filename": fname,
+                "checksum": "00000000000000000000000000000000",
+                "extra": {"image": {"arch": "x86_64"}},
+                "type_name": tname,
+                "build_id": 1234,
+                "nvr": nvr,
+            }
+        )
+
+    fake_koji.insert_archives(archives=archives, build_nvr=nvr)
+
+    with source:
+        items = list(source)
+
+    vmi_items = [i for i in items if isinstance(i, VMIPushItem)]
+
+    for push_item in vmi_items:
+        assert isinstance(push_item, AmiPushItem)
```

Comment:
It seems like at least one of these tests should be doing a bit more in-depth test of the generated push items, rather than just `isinstance`.

This test would pass even if the errata source generated AmiPushItem instances with nonsense in all the fields, wouldn't it?

Reply:
Added more checks to it, was able to see I wasn't setting origin correctly. So it's been updated.